src/vllm_tgis_adapter/http.py

```diff
@@ -63,7 +63,7 @@ async def set_correlation_id(request: Request, call_next: Callable) -> Response:
     }
     serve_kwargs.update(uvicorn_kwargs)
 
-    shutdown_coro = await serve_http(app, **serve_kwargs)
+    shutdown_coro = await serve_http(app, sock=None, **serve_kwargs)
```

Comment:
Let's keep it backwards compatible with 0.7.2 for the time being:

```suggestion
    from vllm.version import __version_tuple__ as vllm_version_tuple
    ...
    if vllm_version_tuple >= (0, 7, 3):
        serve_kargs['sock'] = None
        
    shutdown_coro = await serve_http(app, **serve_kwargs)
```